src/NServiceBus.Core/Sagas/SagaMetadata.cs

```diff
@@ -37,6 +38,8 @@ public partial class SagaMetadata
         }
     }
 
+    internal ISagaNotFoundHandlerInvocation? NotFoundHandler { get; }
```

Comment:
Could we assign a default no-op handler and make it non-nullable?

Reply:
good idea, done